### PR TITLE
Don't trim slash on the end of alias path

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #67

### CODE REVIEW

File: tests/AliasesTest.php [hunk 1]
```diff
@@ -73,8 +107,8 @@ public function testSetTrimsTrailingSlashes(): void
             '@backward' => 'backward\\\\',
         ]);
 
-        $this->assertSame('forward', $aliases->get('@forward'));
-        $this->assertSame('backward', $aliases->get('@backward'));
```
Comment (cebe) [inline tests/AliasesTest.php:77]: I'd keep this check too.

File: tests/AliasesTest.php [hunk 2]
```diff
@@ -10,6 +10,40 @@
...
+            ['test', ['@url' => 'test'], '@url'],
+            ['test/', ['@url' => 'test'], '@url/'],
+            ['test/about', ['@url' => 'test'], '@url/about'],
+            ['test/', ['@url' => 'test/'], '@url'],
+            ['test/', ['@url' => 'test/'], '@url/'],
+            ['test/about', ['@url' => 'test/'], '@url/about'],
+            ['/test', ['@url' => '/test'], '@url'],
+            ['/test/', ['@url' => '/test'], '@url/'],
+            ['/test/about', ['@url' => '/test'], '@url/about'],
+            ['/test/', ['@url' => '/test/'], '@url'],
```
Comment (cebe) [inline tests/AliasesTest.php:31]: even if internally we do not trim slashes, this one should return `/test` and not `/test/`.

```php
$a = $aliases->get('@url') . '/subpath'; 
// expect $a to be "/test/subpath" and not "/test//subpath"
```

**Review by cebe — CHANGES_REQUESTED** [review]
I'd fix trim of slashes only for the edge case of a single slash and not remove it completely. In most cases it is better to trim.

File: tests/AliasesTest.php (hunk 1 above)
Comment (vjik) [reply tests/AliasesTest.php:77]: This PR change this behavior: now if we get only alias, then path returns as is.

**Comment by vjik** [comment]
> I'd fix trim of slashes only for the edge case of a single slash and not remove it completely. In most cases it is better to trim.

Case `/` equal to `subdir/`. If set alias `@url` as `https://example.com/subdir/` then expected result of `$aliases->get($url)` will be ``https://example.com/subdir/`, not `https://example.com/subdir`.

File: tests/AliasesTest.php (hunk 2 above)
Comment (vjik) [reply tests/AliasesTest.php:31]: Hm. But if alias will be `/` then:

```
$a = $aliases->get('@url') . '/subpath'; 
// "//subpath"
```

May be don't change behavior and close this PR? =)

Comment (vjik) [reply tests/AliasesTest.php:31]: > Hm. But if alias will be `/` then:
> 
> ```
> $a = $aliases->get('@url') . '/subpath'; 
> // "//subpath"
> ```
> 
> May be don't change behavior and close this PR? =) Now I like this choice.

**Comment by vjik** [comment]
https://github.com/yiisoft/aliases/issues/67#issuecomment-1823894588
